docs/sql-data-sources-csv.md

```diff
@@ -109,9 +109,9 @@ Data source options of CSV can be set via:
     <td>read</td>
   </tr>
   <tr>
-    <td><code>inferDate</code></td> 
+    <td><code>inferDate</code></td>
```

Comment:
qq are we going to keep the name as is? If we parse them as dates (or timestamps) when user specify the schema, I think it makes more sense to rename it to `prefersDate`

Reply:
Yes, let's change the name to `prefersDate` since we are not going to throw an exception when the schema inference is disabled.

Reply:
Personally, I would prefer `preferDate` instead of `prefersDate` but the latter would be consistent with JSON option `prefersDecimal` so it is fine.